android/src/main/java/com/axmall/asynchttp/AsyncHttpAndroid.java

```diff
@@ -90,15 +106,30 @@ public void clearCookies() {
         this.cookieStore.clear();
     }
 
+    private void clearHeaders() {
```

Comment:
Can we rename this to perhaps `resetHeaders`? `clearHeaders` would imply to me that all of them are removed, while reset could mean an initial state of sorts.